package.json

```diff
@@ -6,8 +6,8 @@
     "build": "$npm_execpath turbo run build",
     "clean": "$npm_execpath turbo run clean",
     "dev": "$npm_execpath turbo run dev",
-    "lint": "$npm_execpath turbo run lint",
-    "lint:fix": "$npm_execpath turbo run lint:fix",
+    "lint": "(pnpm prettier . --check || true) && $npm_execpath turbo run lint",
```

Comment:
I think it's better to fail fast here - also this will also consume the prettier errors. Could you remove pls?

Reply:
Also in this case we want to run `lint` in all workspaces using `turbo` so this command does not need to be adjusted, it's the workspace commands that need to be changed